src/plugins/googledrive/server/services/welcome-service.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = ({ strapi }) => ({
+  getWelcomeMessage() {
+    return 'Welcome to googledrive upload 🚀';
```

Comment:
Is this for the admin dashboard?

Reply:
No, it is to test the successful connection with the google drive plugin.